boot/boot_serial/src/boot_serial.c

```diff
@@ -216,7 +217,8 @@ bs_upload(char *buf, int len)
 
     memset(img_data, 0, sizeof(img_data));
     cbor_buf_reader_init(&reader, (uint8_t *)buf, len);
-    cbor_parser_init(&reader.r, 0, &parser, &value);
+    //cbor_parser_init(&reader.r, 0, &parser, &value);
```

Comment:
Why was this commented out and not removed?

Reply:
obviously I mess with Mynewt here, thx for this comment!